pkg/arvo/lib/base64.hoon

```diff
@@ -59,7 +59,7 @@
     'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
   ::
   |^  |=  bs=octs  ^-  cord
-      =/  [padding=@ blocks=(list word24)]
+      =+  ^-  [padding=@ blocks=(list word24)]
```

Comment:
This looks like an accidental revert. It can be fixed without pushing new pills.

Reply:
Huh, I didn't realize that works.

Reply:
See #2114